praetorian_cli/handlers/chariot.py

```
@@ -1,6 +1,46 @@
+            except ValueError:
+                click.echo(f'Error: invalid input: {raw}')
+                raise SystemExit(1)
+            if choice < 1 or choice > len(account_names):
+                click.echo(f'Error: must be between 1 and {len(account_names)}')
+                raise SystemExit(1)
+            sdk.accounts.assume_role(account_names[choice - 1])
+        else:
+            keychain.account = saved_account
+    except (KeyboardInterrupt, click.Abort, SystemExit):
+        raise SystemExit(0)
+    except Exception:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)